packages/website/src/features/Packages/Function.tsx

```diff
@@ -113,10 +114,10 @@ export const Function: FC<{
     } catch (e) {
       if (!suppressError) {
         try {
-          const provider = new ethers.providers.JsonRpcProvider(
-            publicClient.chain.rpcUrls.public.http[0] as string
-          );
-          await handleTxnError(cannonOutputs, provider, e);
+          // const provider = new ethers.providers.JsonRpcProvider(
+          //   publicClient?.chain?.rpcUrls?.public?.http[0] as string
+          // );
+          // await handleTxnError(cannonOutputs, provider, e);
           console.error(e);
         } catch (e2) {
```

Comment:
I think we don't need the `try catch` here anymore.

Reply:
I know but I kept it to fix the issue with handleTxError. For some reason we cannot use that function in the react package